main/qml_frontend.go

```diff
@@ -170,7 +170,7 @@ func (t *qmlfrontend) onNew(v *backend.View) {
 			return
 		}
 		item.Set("myView", fv)
-		item.Set("fontSize", float64(v.Settings().Get("font_size", 12).(int)))
+		item.Set("fontSize", float64(12))
```

Comment:
This was because `v.Settings().Get("font_size", 12)` is sometimes returning int sometimes float64.
A better solution added by 6b5cc64